_DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml

```diff
@@ -17,16 +17,11 @@
   connection: local
   tasks:
     - name: Remove local files
-      local_action: file path={{ item }} state=absent
-      with_items:
-        - ./openrc
-        - ./admin-ciaorc
-        - ./demo-ciaorc
-        - ./clouds.yaml
-        - ./certificates
-        - ./images/.cnci_ready
-        - ./fetch
-        - /tmp/go
+      file: path={{ item }} state=absent
+      with_fileglob:
+        - /tmp/go/*
```

Comment:
Ansible creates `{{ gopath }}` which defaults to /tmp/go if the directory doesn't exist.

The workflow is if you dont have `{{ gopath }}` created, ansible creates the directory.
Ansible pulls ciao from github with `go get -v` not `-u`.

This allows the admin to`cd {{gopath}}` `git checkout -b myitem` `hack hack`
run ansible and test results.